scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+    use radio modem bandwidth and fit with user applications.
+- group: solution
+  name: dgnss_solution_mode
+  type: enum
+  units:
+  default value:
+  enumerated possible values: Low Latency, Time Matched
+  Description: Determines the type of RTK solution which will be output.
+  Notes: A low latency solution uses an internal model of anticipated satellite observations
+    to provide RTK output with minimal latency but slightly reduced accuracy.  This
+    mode should be used for most applications.  Where accuracy is desired with the
+    expectation of increased latency, the "Time matched" mode can be chosen.  This
+    means that the RTK output will require a corresponding set of correction observations
+    for each timestamp.
+- group: solution
+  name: dgnss_filter
+  type: enum
+  units:
+  default value: Fixed
+  enumerated possible values: Fixed,Float
+  Description: Determines the type of IAR the pikis will attempt to achieve.
+  Notes: If "fixed", the Piksi will output a fixed integer ambiguity estimate.  If
+    no fixed solution is available, it will rever to the float solution.  If "float",
+    the device will only output the float integer ambiguity estimate.
+- group: solution
+  name: known_baseline_n
+  type: double
+  units: meters (north)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: solution
+  name: known_baseline_e
+  type: double
+  units: meters (east)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: solution
+  name: known_baseline_d
+  type: double
+  units: meters (down)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: system_info
+  name: serial_number
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: The serial number of the Piksi
+  Notes: This number should match the number on the barcode and cannot be modified
+- group: system_info
+  name: firmware_version
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Indicates the firmware version for the Local Piksi
+  Notes: 'For user generated firmware, this information will appear the same as the
+    GIT command: "git describe --dirty"'
+- group: system_info
+  name: firmware_built
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Date of hardware build
+  Notes:
+- group: system_info
+  name: hw_revision
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: hardware revision for Piksi
+  Notes:
+- group: system_info
+  name: nap_version
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: Version of the Navigation Acceleration Process (NAP)
```

Comment:
haha, I think we never really agreed on what "NAP" stood for. I guess this is where the rubber finally meets the road.

Unless we just say "Version of the SwiftNAP FPGA firmware"